### PR TITLE
BT-28951: Added aria label for main menu search form input

### PR DESCRIPTION
**Pull request by M-Barz**

### CODE REVIEW

File: Services/Search/classes/class.ilMainMenuSearchGUI.php [hunk 1]
```diff
@@ -103,6 +103,7 @@ public function getHTML()
         $this->tpl->setVariable('FORMACTION', 'ilias.php?baseClass=ilSearchController&cmd=post' .
             '&rtoken=' . $ilCtrl->getRequestToken() . '&fallbackCmd=remoteSearch');
         $this->tpl->setVariable('BTN_SEARCH', $this->lng->txt('search'));
+        $this->tpl->setVariable('SEARCH_INPUT_LABEL', mb_strtolower($this->lng->txt('search_field')));
```
Comment (smeyer-ilias) [inline Services/Search/classes/class.ilMainMenuSearchGUI.php]: Why strtolower?
If you want to use mb_strtolower in other case please use the wrapper ilStr::strToLower in ./Services/Utilties
The mb_ methods can only be used, if the multibyte extension is loaded, which is not a required module in our installation guide.

File: lang/ilias_de.lang [hunk 2]
```diff
@@ -4909,6 +4909,7 @@ common#:#search_active#:#Aktive Benutzer
 common#:#search_at_current_position#:#An der aktuellen Position
 common#:#search_engine#:#Lesbare .html-URLs
 common#:#search_for#:#Suchen
+common#:#search_field#:#Sucheingabefeld
```
Comment (smeyer-ilias) [inline lang/ilias_de.lang]: Whenever possible avoid the common module. You could can use the search module here.

File: lang/ilias_en.lang [hunk 3]
```diff
@@ -3478,6 +3478,7 @@ common#:#scorm_new#:#Create SCORM Learning Module (Authoring Mode)
 common#:#search_active#:#Include active users
 common#:#search_engine#:#Readable .html URLs
 common#:#search_for#:#Search For
+common#:#search_field#:#Search input field
```
Comment (smeyer-ilias) [inline lang/ilias_en.lang]: Please use uppercase in english (if not a completre sentence): "Search Input Field"

**Review by smeyer-ilias — CHANGES_REQUESTED** [review]

**Comment by smeyer-ilias** [comment]
Please can you add a link to the mantis report.

**Comment by M-Barz** [comment]
Mantis Report:
https://mantis.ilias.de/view.php?id=28951

**Comment by M-Barz** [comment]
Implemented the requested Changes :)

File: lang/ilias_en.lang [hunk 4]
```diff
@@ -6888,6 +6887,7 @@ sahs#:#sahs_sco_objective#:#Objectives
 sahs#:#sahs_sequencing#:#Sequencing
 sahs#:#sahs_unit#:#SCO
 search#:#add_members_header#:#Add Members
+search#:#search_field#:#Search input field
```
Comment (smeyer-ilias) [inline lang/ilias_en.lang]: "Please use uppercase in english (if not a complete sentence): "Search Input Field"

**Review by smeyer-ilias — CHANGES_REQUESTED** [review]
Please fix the english language variable.
After this change please commit and merge to changes to 5.4, 6 and trunk

**Pull request merged**